### PR TITLE
Update new-netnat.md

### PR DESCRIPTION
**Pull request by sukihanabi**
#360 
Action Taken: 
Added example for -ExternalIPInterfaceAddressPrefix "a.b.c.0/24"
New-NetNat -Name "AllTenants" -ExternalIPInterfaceAddressPrefix "10.0.0.0/24"

References:
Set up a NAT network 
https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/user-guide/setup-nat-network

http://www.checkyourlogs.net/?p=58493
http://techgenix.com/nat-network-hyper-v-vms/

Screenshot: New-Netnat
https://snag.gy/13M9rf.jpg

Get-nat
https://snag.gy/T6dZF2.jpg

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: docset/windows/netnat/new-netnat.md [hunk 1]
```diff
@@ -56,6 +56,12 @@ PS C:\> New-NetNat -Name "AllTenants" -ExternalIPInterfaceAddressPrefix "a.b.c.0
 This command creates a NAT object named AllTenants for all the computers in the specified subnet.
 This example uses the placeholder a.b.c.0/24 to represent a public Internet address prefix.
 
+From the above, for this example, we'll use 10.0.0.0/24.
```
Comment (JohanFreelancer9) [inline docset/windows/netnat/new-netnat.md]: From the above, 10.0.0.0/24 will be used as an example.

File: docset/windows/netnat/new-netnat.md [hunk 2]
```diff
@@ -56,6 +56,12 @@ PS C:\> New-NetNat -Name "AllTenants" -ExternalIPInterfaceAddressPrefix "a.b.c.0
...
+For our example, run the following to setup the NAT network:
```
Comment (JohanFreelancer9) [inline docset/windows/netnat/new-netnat.md]: For **this** example, run the following to **set up** the NAT network:

Please do changes as indicated in bold.

Comment (sukihanabi) [reply docset/windows/netnat/new-netnat.md]: @johanfreelance9 - commit changes as advised

**Pull request merged**